lib/wallet/src/Cardano/Wallet/DB/Sqlite/Schema.hs

```diff
@@ -492,12 +498,16 @@ CBOR
     deriving Show Generic Eq
 
 Submissions
-    submissionTxId TxId  sql=tx_id
-    submissionTx W.SealedTx sql=tx
-    submissionExpiration SlotNo sql=expiration
-    submissionAcceptance (Maybe SlotNo) sql=acceptance
-    submissionWallet W.WalletId sql=wallet_id
-    submissionStatus TxSubmissionStatusEnum sql=status
+    submissionTxId                  TxId                sql=tx_id
+    submissionTx                    W.SealedTx          sql=tx
+    submissionExpiration            SlotNo              sql=expiration
+    submissionAcceptance            (Maybe SlotNo)      sql=acceptance
+    submissionWallet                W.WalletId          sql=wallet_id
+    submissionStatus                TxSubmissionStatusEnum sql=status
+    submissionMetaDirection         W.Direction         sql=direction
+    submissionMetaSlot              SlotNo              sql=slot
+    submissionMetaBlockHeight       BlockHeight         sql=block_height
+    submissionMetaAmount            W.Coin              sql=amount
```

Comment:
```suggestion
    submissionMetaSlot              SlotNo              sql=slot
    submissionMetaBlockHeight       BlockHeight         sql=block_height
    submissionMetaAmount            W.Coin              sql=amount
    submissionMetaDirection         W.Direction         sql=direction
```

Small reordering to move `direction` and `amount` closer together, as they are conceptually related.